DTS-STN/Alpha-Site
# update splash page

# Description

[Update splash page](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities/?workitem=67506)

Update the splash page to match the new design on [Figma](https://www.figma.com/file/ke3SrZNGMP3AjwzMMuEgbo/%5Bcurrent%5D-BDM-DECD-SC-Labs?node-id=10436%3A27681)

## Acceptance Criteria

Splash page matches the new design on Figma

## Test Instructions

1. yarn dev
2. visit the index page `localhost:3000/`, if you are being redirected to the home page, make sure to clear cookie
3. Make sure the new splash page looks like the design on Figma.
